packages/core/src/App/Containers/RealAccountSignup/modal-login-prompt.jsx

```diff
@@ -16,7 +14,11 @@ const ModalLoginPrompt = () => {
 
             <div>
                 <Button secondary text={localize('Log In')} onClick={() => redirectToLogin(false, getLanguage())} />
-                <Button primary text={localize('Sign Up')} onClick={() => redirectToSignUp({ is_deriv_crypto })} />
+                <Button
+                    primary
+                    text={localize('Sign Up')}
+                    onClick={() => redirectToSignUp({ is_deriv_crypto: false })}
```

Comment:
It is not needed to pass any option to this func.

Reply:
Thanks @matin-deriv for highlighting this. 
I have removed this as you mentioned.